plugins/BEdita/API/src/Controller/AppController.php

```diff
@@ -55,7 +55,7 @@ public function initialize()
 
         $this->response = $this->response->withHeader('X-BEdita-Version', Configure::read('BEdita.version'));
 
-        $this->loadComponent('BEdita/API.Paginator');
+        $this->loadComponent('BEdita/API.Paginator', Configure::read('Pagination'));
```

Comment:
If no `Pagination` key is found in config an empty array must be passed

(see travis errors)

Also a sample setting in `app.default.php` is missing